frontend/src/metabase/admin/permissions/selectors/data-permissions/breadcrumbs.ts

```diff
@@ -91,7 +97,9 @@ export const getGroupsDataEditorBreadcrumbs = (
   const hasMultipleSchemas = database.schemasCount() > 1;
 
   if (tableId == null) {
-    return [databaseItem, hasMultipleSchemas && schemaItem].filter(isNotNull);
+    return [databaseItem, hasMultipleSchemas && schemaItem].filter(
+      isNotNull,
+    ) as EditorBreadcrumb[];
```

Comment:
Can we make `databaseItem` etc to be `EditorBreadcrumb`?

Reply:
I had to cast types here because TypeScript doesn't believe that running `filter(Boolean)` or `filter(isNotNull)` will exclude falsey values :(

Reply:
hmm actually `isNotNull` should do the trick

Reply:
@ranquild think I've fixed it, mind giving another look at [9490ea9](https://github.com/metabase/metabase/pull/30331/commits/9490ea983c432177171e2ae2399b353d2276963c)?

Reply:
@kulyk maybe add another helper like `isNotFalsy`?

Reply:
Oh, I like it more, thanks 👍

Reply:
Done in [972d616](https://github.com/metabase/metabase/pull/30331/commits/972d6160fda44637cd068986246c617fb8e95cac)